juntagrico/entity/member.py

```diff
@@ -74,6 +74,25 @@ def active_shares(self):
         """
         return self.share_set.filter(paid_date__isnull=False).filter(payback_date__isnull=True)
 
+    def active_shares_for_date(self, date=timezone.now):
+        return self.share_set.filter(paid_date__lte=date).filter(Q(payback_date__isnull=True) | Q(payback_date__gte=date))
+
+    @property
+    def active_share_years(self):
```

Comment:
diese methode macht 3 datenbank abrufe (immer wenn [0] in der zeile steht) vorallem werden jedesmal alle anteilscheine rübergeladen.
da ein member nicht unendlich anteilscheine hat (normalerweise 1 bis 2) und im extremfall 50 oder so könnte man das ev in einem foorloop ausrechnen?